src/components/DatePicker/DatePicker.tsx

```diff
@@ -141,6 +141,10 @@ const DatePicker: React.FC<DatePickerProps & TestProps> = ({
         setIsOpen(false);
         setRange(EMPTY_STATE);
 
+        if (filterConfig?.filterType === 'added') {
+          onClear();
+        }
+
```

Comment:
when we click the X icon on Filter DatePicker, we have 2 inner callbacks now: `onClear` and `onChange(EMPTY_STATE)`. are we sure we want this?

Reply:
Fixed with https://github.com/Orfium/orfium-ictinus/pull/854/commits/149839573bce15f8a5c4ad801dd47f6d47b05ab1